.github/workflows/gcp-test-deploy.yml

```
@@ -0,0 +1,315 @@
+      #
+      # If we change the state path without changing the version,
+      # this job will take a few hours, because it will do a full rebuild.
+      - name: Get specific changed files
+        id: changed-files-specific
+        uses: tj-actions/changed-files@v19
+        with:
+          files: |
+            zebra-state/**/constants.rs
+
+      - name: Inject slug/short variables
+        uses: rlespinasse/github-slug-action@v4
+        with:
+          short-length: 7
+
+      - name: Downcase network name for disks
+        run: |
+          NETWORK_CAPS=${{ inputs.network }}
+          echo "NETWORK=${NETWORK_CAPS,,}" >> $GITHUB_ENV
+
+      # Setup gcloud CLI
+      - name: Authenticate to Google Cloud
+        id: auth
+        uses: google-github-actions/auth@v0.7.1
+        with:
+          workload_identity_provider: 'projects/143793276228/locations/global/workloadIdentityPools/github-actions/providers/github-oidc'
+          service_account: 'github-service-account@zealous-zebra.iam.gserviceaccount.com'
+          token_format: 'access_token'
+
+      - name: Create GCP compute instance
+        id: create-instance
+        if: (${{ inputs.needs_constants }} && steps.changed-files-specific.outputs.any_changed == 'true' && github.event_name == 'push') || ${{ inputs.${{ inputs.test_id }}s }} || !${{ inputs.needs_constants }}
+        run: |
+          gcloud compute instances create-with-container "${{ inputs.test_id }}-${{ env.GITHUB_REF_SLUG_URL }}" \
+          --boot-disk-size 100GB \
+          --boot-disk-type pd-ssd \
+          --create-disk name="${{ inputs.test_id }}-${{ env.GITHUB_SHA_SHORT }}",device-name="${{ inputs.test_id }}-${{ env.GITHUB_SHA_SHORT }}",size=100GB,type=pd-ssd \
+          --container-mount-disk mount-path="${{ inputs.zebra_state_path }}",name="${{ inputs.test_id }}-${{ env.GITHUB_SHA_SHORT }}" \
+          --container-image debian:buster \
+          --container-restart-policy=never \
+          --machine-type ${{ env.MACHINE_TYPE }} \
+          --scopes cloud-platform \
+          --metadata=google-monitoring-enabled=true,google-logging-enabled=true \
+          --tags ${{ inputs.app_name }} \
+          --zone "${{ env.ZONE }}"
+          sleep 30
+
+      - name: ${{ inputs.test_name }}
+        id: ${{ inputs.test_id }}
+        if: ${{ steps.create-instance.outcome == 'success' }}
+        run: |
+          gcloud compute ssh \
+          ${{ inputs.test_id }}-${{ env.GITHUB_REF_SLUG_URL }} \
+          --zone ${{ env.ZONE }} \
+          --quiet \
+          --ssh-flag="-o ServerAliveInterval=5" \
+          --command \
+          "docker run -e ${{ inputs.test_variable }}=1 -t --name ${{ inputs.test_id }} \
+          --mount type=bind,source=/mnt/disks/gce-contaißners-mounts/gce-persistent-disks/${{ inputs.test_id }}-${{ env.GITHUB_SHA_SHORT }},target=${{ inputs.zebra_state_path }} \
+          ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:sha-${{ env.GITHUB_SHA_SHORT }}
```

Bug: missing end quote

```suggestion
          ${{ env.GAR_BASE }}/${{ env.IMAGE_NAME }}:sha-${{ env.GITHUB_SHA_SHORT }}"
```

---

Fixed by: [fix(rw): minor fixes](https://github.com/ZcashFoundation/zebra/pull/4271/commits/044e944715ff59bccd0eb2f415f9a4737cfb336a)